docs/tutorials/11_quadratic_hamiltonian_and_slater_determinants.ipynb

```diff
@@ -0,0 +1,454 @@
+      "text/plain": [
+       "<Figure size 1544.38x264.88 with 1 Axes>"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit_nature.circuit.library import FermionicGaussianState\n",
+    "\n",
+    "occupied_orbitals = (0, 2)\n",
+    "eig = np.sum(orbital_energies[list(occupied_orbitals)]) + transformed_constant\n",
+    "print(\"Eigenvalue:\")\n",
+    "print(eig)\n",
+    "\n",
+    "circuit = FermionicGaussianState(\n",
+    "    transformation_matrix, occupied_orbitals=occupied_orbitals\n",
+    ")\n",
+    "circuit.draw(\"mpl\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
```

Comment:
Also this cell appears quite detached from the section of the text which
discusses the mapper. This could be reshuffled slightly to improve things.

You could even consider printing the qubit operator rather than just asserting
its content.

Reply:
I've added a text section introducing this cell.